components/OtherUserSpectrum.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <li>
```

Comment:
the choice of using an `li`as outer tag, limit the versatility of this component, since it now can only be used as a child of a `ul`.
More elegant would be to make this a more standalone and versatile component with div as outer tag. Then it's parent component can take care of the rendering of the this child component as a list.

Naming wise, there is nothing inherent in the component that makes this about an "OtherUser", so I'd prefer something like "SpectrumInline" or "SpectrumItem". It can then be used in the ch3 `ChapterOpening` to render an OtherUser's profile, or elsewhere to render another (set of) inline spectrum/a.

It might make sense to have two components that are semantically associated (and reflect this in their naming):
- `SpectrumItem`
- `SpectrumItemList`